packages/main/src/plugin/configuration-registry.ts

```diff
@@ -121,6 +129,17 @@ export class ConfigurationRegistry implements IConfigurationRegistry {
       configData = {};
     }
     this.configurationValues.set(CONFIGURATION_DEFAULT_SCOPE, configData);
+
+    // Load managed defaults
+    const defaultConfiguration = new DefaultConfiguration();
+    const defaults = await defaultConfiguration.getContent();
+    this.configurationValues.set(CONFIGURATION_SYSTEM_MANAGED_DEFAULTS_SCOPE, defaults);
```

Comment:
If there is an error in the config written, nothing will catch it

Reply:
I'm wondering if we want to validate the content deeply, or just enough to ensure it will work, like checking at least this is an Object with entries